services/pubsub.js

```diff
@@ -2,4 +2,13 @@ const {RedisPubSub} = require('graphql-redis-subscriptions');
 
 const {connectionOptions} = require('./redis');
 
-module.exports = new RedisPubSub({connection: connectionOptions});
+let pubsub = null;
+module.exports = () => {
```

Comment:
Maybe exporting a object with a descriptive function is more clear:

```js
// pubsub.js
module.exports = {
  getInstance() {
    if (pubsub) {
      return pubsub;
     }
    pubsub = new RedisPubSub({connection: connectionOptions});
  },
}
```

So it can be consumed like this:

```js
const pubsub = require('pubsub');
const pubsubInstance = pubsub.getInstance();
```

Instead of

```js
const pubsub = require('pubsub');
const pubsubInstance = pubsub();
```
The code `pubsub = pubsub()` as used in this PR, was not immediately clear to me, what it meant.